java/query/src/main/java/sleeper/query/model/output/S3ResultsOutput.java

```diff
@@ -102,6 +102,8 @@ public ResultsOutputInfo publish(Query query, CloseableIterator<Record> results)
             LOGGER.info("Wrote {} records to {} in {} seconds (rate of {})",
                     count, outputFile, durationInSeconds, rate);
             return new ResultsOutputInfo(count, Collections.singletonList(outputLocation));
+        } catch (RuntimeException e) {
+            throw e;
```

Comment:
This will skip the logging and return call below. Maybe the catch(Exception e) below should be catch(RuntimeException | IOException e)?